### DISCUSSION TITLE
What windows 2 in 1 machines are fully working with brunch?

### DISCUSSION THREAD

**Discussion by juan9999** [sebanc/brunch#1298] [General]
I'd like a dual boot windows/chrome os 2 in 1 machine that has a great small screen (ideally 3:2 ratio) that's not crazy expensive. Any suggestions?

**Comment by petm5** [comment]
Anything that works with Linux and supports virtualization technology should work fine but you might need to mess around with firmware and/or drivers for some (rare-ish) hardware.

**Comment by tejasraman** [comment]
Acer Spin 5 (2020) is great - 3:2, active pen in Linux/Brunch, and is a 2-in-1

[https://sciactive.com/2020/12/04/how-to-install-ubuntu-on-acer-spin-5-sp513-54n-for-the-perfect-linux-2-in-1/](https://sciactive.com/2020/12/04/how-to-install-ubuntu-on-acer-spin-5-sp513-54n-for-the-perfect-linux-2-in-1/) how to set the BIOS properly for Linux/Brunch - Acer makes the default settings painful for Linux

**Reply by juan9999** [reply]
thanks. is this the 2020 model?

https://www.newegg.com/acer-spin-5-grey/p/1XV-000G-00828?Description=SP513-52N-5621&cm_re=SP513-52N-5621-_-1XV-000G-00828-_-Product&quicklink=true

**Reply by tejasraman** [reply]
This one is 2020 - this is not the best deal on one, there are better prices on it: [https://www.staples.com/acer-spin-5-sp513-54n-56m2-13-5-refurbished-laptop-intel-i5-16gb-memory-512gb-ssd-windows-10/product_24497048?ci_sku=24497048&KPID=24497048&cid=PS:GS:SBD:PLA:Tech&gclid=Cj0KCQiA_c-OBhDFARIsAIFg3ewnC99zYUn2Wi_l-77KPNSAyHwccwoyMsa2KWTUiqZE3p4es0rFZSMaAnv1EALw_wcB](https://www.staples.com/acer-spin-5-sp513-54n-56m2-13-5-refurbished-laptop-intel-i5-16gb-memory-512gb-ssd-windows-10/product_24497048?ci_sku=24497048&KPID=24497048&cid=PS:GS:SBD:PLA:Tech&gclid=Cj0KCQiA_c-OBhDFARIsAIFg3ewnC99zYUn2Wi_l-77KPNSAyHwccwoyMsa2KWTUiqZE3p4es0rFZSMaAnv1EALw_wcB)

The one you linked is 2018 model with an 8th gen i5 and a 16:9 display

**Reply by tejasraman** [reply]
Here is Acer’s website: [https://www.acer.com/ac/en/US/content/series/spin5](https://www.acer.com/ac/en/US/content/series/spin5)
**Be warned: You need to enable AHCi and PS/2 in BIOS and, if you plan to use Windows, reinstall it.**

**Reply by tejasraman** [reply]
Acer’s website shows the 2021 Spin 5, not the 2020. There are some new ones on Amazon and Newegg. Pen should work with brunch too in both spin3 and spin5

**Comment by tejasraman** [comment]
You could also get the similar Acer Spin 3: [https://acerrecertified.com/acer-spin-3-14-laptop-intel-core-i5-1035g1-1ghz-8gb-ram-256gb-ssd-windows-10-home-sp314-54n-58q7/](https://acerrecertified.com/acer-spin-3-14-laptop-intel-core-i5-1035g1-1ghz-8gb-ram-256gb-ssd-windows-10-home-sp314-54n-58q7/) on Acer’s official refurbished site

or on Newegg: [https://www.newegg.com/p/1XV-000G-00262?Description=Spin%203&cm_re=Spin_3-_-9SIAMRPDH61266-_-Product&quicklink=true](https://www.newegg.com/p/1XV-000G-00262?Description=Spin%203&cm_re=Spin_3-_-9SIAMRPDH61266-_-Product&quicklink=true)

Same procedure as the Spin 5, same BIOS settings, same Brunch install.
**Unlike Spin 5, this does not have a 3:2 ratio**
